File: Dynamic Programing/Fibonacci/PHP/fibonacci_in_iterative.php
<!DOCTYPE html>
<html>
<head>
</head>
<body>

<form action="fibonacci_in_iterative.php" method="post">
<input type="number" name="number">
<input type="submit" value="=">
</form>

<?php

function fib($n){
	
	$data = array(0,$n+1);
	$data[0] = 0;
	$data[1] = 1;
	for($i=2; $i<=$n; $i++){
		$data[$i] = $data[$i-1] + $data[$i-2];
	}
	return $data[$n];
}

if(isset($_POST["number"])){
	$n = $_POST["number"];
	echo fib($n);
}


?>

</body>
</html>

File: Dynamic Programing/Fibonacci/C++/fibonacci_in_iterative.cpp
#include <iostream>
using namespace std;
int fib(int n)
{
    int f[n];
    f[0] = 0;
    f[1] = 1;

    for(int i=2; i<=n; i++){
        f[i] = f[i-1] + f[i-2];
    }
    return f[n];
}
int main()
{
    int n;
    while(true){
        cin >> n;
        if(n <= 0) break;
        cout << fib(n) << endl;
    }
    return 0;
}